### PR TITLE
Solid, non-DHT test for Vladimir imitating Ursula and trying to collect funds.

### PR DESCRIPTION
**Pull request by jMyles**
* Character re-organization.
* `Vladimir` class, with logic for trying to imitate and defraud Ursula.
* Now that the DHT is pulled, a much more straightforward and comprehensive test of of the validity check is possible - see the changes to `test_network_actors`

### CODE REVIEW

File: nucypher/characters/unlawful.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
+from nucypher.characters.lawful import Ursula
+from nucypher.crypto.powers import CryptoPower
+from nucypher.utilities.sandbox import EvilMiddleWare
```
Comment (michwill) [inline nucypher/characters/unlawful.py]: Very nice name!

File: nucypher/config/constants.py [hunk 2]
```diff
@@ -1,28 +1,3 @@
...
-DEFAULT_DHT_PORT = DEFAULT_REST_PORT + 100
-DEFAULT_DB_NAME = "ursula.{port}.db".format(port=DEFAULT_REST_PORT)
-
-DEFAULT_SEED_NODE_DIR = os.path.join(DEFAULT_CONFIG_ROOT, 'seed_nodes')
-DEFAULT_KNOWN_NODE_DIR = os.path.join(DEFAULT_CONFIG_ROOT, 'known_nodes')
-
-DEFAULT_SIMULATION_PORT = 5555
-DEFAULT_SIMULATION_REGISTRY_FILEPATH = os.path.join(DEFAULT_CONFIG_ROOT, 'simulation_registry.json')
+DEFAULT_CONFIG_FILE_LOCATION = "cool winnebago"
+DEFAULT_CONFIG_ROOT = "my uncle has one just like it."
```
Comment (michwill) [inline nucypher/config/constants.py]: Is this going to be replaced with something?

**Comment by KPrasch** [comment]
Good Question - At the moment the answer is No.  You have to manually initialize the config with the CLI before running a node; however, I am working on a development mode that will automatically use a temporary storage area during the node's runtime.

File: nucypher/config/constants.py (hunk 2 above)
Comment (jMyles) [reply nucypher/config/constants.py]: Yes, most certainly.  :-)

**Pull request merged**